src/list.tsx

```diff
@@ -407,13 +394,18 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
     } = props;
 
     const radioContext = useRadioContext();
-    const disabled = props.disabled || (props.radioValue !== undefined && radioContext.disabled);
+    const disabled =
+        props.disabled || (areRadioRowContentProps(props) !== undefined && radioContext.disabled);
```

Comment:
`areRadioRowContentProps` returns a boolean, why are you comparing with `undefined`?
shouldn't this be something like this?

```suggestion
        props.disabled || (areRadioRowContentProps(props) && radioContext.disabled);
```